### PR TITLE
- added react-router-4

### LINKED ISSUE #3: Use React Router 4

### PR DESCRIPTION
**Pull request by JesseCarlos**
Fixes #3

### CODE REVIEW

File: imports/ui/Routes.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { render } from 'react-dom';
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import Home from './Home';
+import TasksContainer from '/imports/containers/TasksContainer';
+
+export default Routes = () => {
+    return (
+        <div>
```
Comment (filipenevola) [inline imports/ui/Routes.js]: why this div?

File: imports/ui/Menu.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Drawer from 'material-ui/Drawer';
+import MenuItem from 'material-ui/MenuItem';
+import { Link } from 'react-router-dom';
+
+export default class Menu extends React.Component {
+    render() {
+        return (
+            <div>
```
Comment (filipenevola) [inline imports/ui/Menu.js]: why this div?

File: imports/ui/Home.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Card, CardText } from 'material-ui/Card';
+
+export default Home = () => {
+    return (
+        <div>
```
Comment (filipenevola) [inline imports/ui/Home.js]: div again hehe

File: imports/ui/App.js [hunk 4]
```diff
@@ -1,15 +1,32 @@
...
-    <AppBar title="Meteor React Latest" />
-        <div>
-            <TasksContainer/>
-        </div>
-    </div>
-);
\ No newline at end of file
+    render() {
+        return (
+            <div>
```
Comment (filipenevola) [inline imports/ui/App.js]: div....?

File: imports/ui/Routes.js (hunk 1 above)
Comment (JesseCarlos) [reply imports/ui/Routes.js]: I did think that is required

File: imports/ui/Menu.js (hunk 2 above)
Comment (JesseCarlos) [reply imports/ui/Menu.js]: Same here

File: imports/ui/Home.js (hunk 3 above)
Comment (JesseCarlos) [reply imports/ui/Home.js]: Same here

File: imports/ui/App.js (hunk 4 above)
Comment (JesseCarlos) [reply imports/ui/App.js]: Here is required a div

**Pull request merged**